README.rst

```diff
@@ -138,10 +138,16 @@ the PSF for inclusion in the documentation.
 
     poedit glossary.po
 
-3. 存檔以後，執行以下列指令編譯輸出文件，以確保你的修改沒有 rST 的語法錯誤或警告 ::
+3. 存檔以後，執行以下列指令編譯輸出完整文件，以確保你的修改沒有 rST 的語法錯誤或警告 ::
 
     VERSION=3.12 make all
 
+   或者只想快速檢查是否有 rST 語法錯誤 ::
```

Comment:
```suggestion
   或者只想快速檢查是否有 reST 語法錯誤 ::
```

Reply:
Just realized that the formal abbreviation is "reST". Thanks for catching this!
[Source](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#restructuredtext-primer)